cmd/james/serve.go

```diff
@@ -144,6 +154,7 @@ func (s *serveCmdOptions) RegisterHandlers() {
 		game.NewMuteCommand(),
 		help.NewHelpCommand(),
 		hive.NewHiveCommandForBob(s.db),
+		game.NewLookCommand(s.db),
```

Comment:
Do we need to install it in james also?

Reply:
Forgot that james isn't the gaming bot, I'll remove that 😛